packages/bpk-component-datatable/src/BpkDataTableColumn.js

```diff
@@ -0,0 +1,41 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { Column } from 'react-virtualized';
+import { cssModules } from 'bpk-react-utils';
+
+import STYLES from './bpk-column.scss';
+
+const getClassName = cssModules(STYLES);
+
+const BpkDataTableColumn = () => null;
```

Comment:
This is interesting - what's the workaround here? I'm assuming it's to get round the prop type issues we discussed in a previous comment, but I think it needs a comment for other contributors so that they understand whats going on.